website/profile/views.py

```diff
@@ -165,14 +164,11 @@ def update_user(auth):
         ############
 
         # get the first email that is set to primary and has an address
-        primary_email = next(
-            (
-                each for each in data['emails']
+        primary_email = next(each for each in data['emails']
```

Comment:
Debating this. The extra parens were to help with syntax formatting, and the prior version looks a little better. But if we end up running `black` over this, it won't matter, I suppose.

Reply:
Not gonna worry about this for now.